src/components/infoWindow.vue

```diff
@@ -80,11 +172,12 @@ export default {
     if (this.$infoWindow) {
       this.$infoWindow.setMap(null);
     }
+    this.$off('marker-ready',this.markerReady);
```

Comment:
nice. i will remove this. i'm affray the list of observer be big because deleted objects. but i will investigate and remove if no side effect.